contributing.md

```diff
@@ -0,0 +1,164 @@
+# Documentation Contributing Guide
+
+We welcome contributions from all users and the community. By contributing, you agree to the [Percona Community code of conduct](https://percona.community/contribute/coc/). Thank you for deciding to contribute and helping us improve the [ProxySQL Admin Tool documentation](https://docs.percona.com/proxysql/index.html).
+
+You can contribute to the documentation in either of the following ways:
+
+
+
+- [Documentation Contributing Guide](#documentation-contributing-guide)
+  - [Add a forum topic](#add-a-forum-topic)
+  - [Request a change with a Jira issue](#request-a-change-with-a-jira-issue)
+  - [Contribute to documentation yourself](#contribute-to-documentation-yourself)
+    - [What you should know](#what-you-should-know)
+    - [What happens after you create the pull request](#what-happens-after-you-create-the-pull-request)
+    - [Edit documentation online with GitHub](#edit-documentation-online-with-github)
+    - [Edit documentation locally](#edit-documentation-locally)
+    - [Building the documentation](#building-the-documentation)
+
+## Add a forum topic
+
+In the [Percona Product Documentation category](https://forums.percona.com/c/percona-product-documentation/71) in the Percona Community Forum, select **New Topic**. Complete the form and select **Create Topic** to add the topic to the forum.
+
+<img src= "./docs/_images/new-topic.png" alt ="create a topic" width = 40% height = 40% />
+
+## Request a change with a Jira issue
+
+If you would rather not [contribute to the documentation yourself](#contribute-to-documentation-yourself) let us know about the issue by adding a Jira ticket. Use the following procedure to create a Jira ticket:
```

Comment:
Add comma after [contribute to the documentation yourself](#contribute-to-documentation-yourself)